Parent/Parent/Assignments/AssignmentDetailsViewController.swift

```diff
@@ -235,3 +219,18 @@ class AssignmentDetailsViewController: UIViewController, CoreWebViewLinkDelegate
         env.router.show(compose, from: self, options: .modal(embedInNav: true), analyticsRoute: "/conversations/compose")
     }
 }
+
+extension AssignmentDetailsViewController {
```

Comment:
What's the reason of this method being in an extension? It's not a protocol method.

Reply:
cleaned it up.